### PR TITLE
feat: homepage actions and federation selector

### PR DESCRIPTION
**Pull request by Kodylow**
Took another pass at improving the home page, it's much better now with a nice selector

image

## Release Notes

- **New Features**
	- Introduced a dynamic navigation header with service selection capabilities.
	- Added modals for connecting, editing, and removing services, enhancing user interaction.
	- Implemented a responsive service table for better service management.
	- Added a component for displaying messages when no services are connected.

- **Improvements**
	- Enhanced localization support with new language entries for modals and UI elements.
	- Improved error handling and user feedback in service connection processes.

- **Refactor**
	- Streamlined component structure for better maintainability and readability.

### CODE REVIEW

File: apps/router/src/index.tsx [hunk 1]
```diff
@@ -14,31 +14,36 @@ import { AppContextProvider } from './context/AppContext';
 import { HomePage } from './home/HomePage';
 import { GuardianContextProvider } from './context/guardian/GuardianContext';
 import { GatewayContextProvider } from './context/gateway/GatewayContext';
+import { Wrapper } from './components/Wrapper';
 
 i18nProvider(languages);
 
+// ... existing imports ...
```
Comment (tvolk131) [inline apps/router/src/index.tsx:21]: AI leftovers? 👀

**Review by tvolk131 — CHANGES_REQUESTED** [review]
Just one small thing, otherwise LGTM and works on my machine!

File: apps/router/src/index.tsx (hunk 1 above)
Comment (Kodylow) [reply apps/router/src/index.tsx:21]: oh lol haha yup fixing

**Review by tvolk131 — APPROVED** [review]
Send it! 🚀

**Pull request merged**